src/integrationTest/resources/application.yaml

```diff
@@ -10,6 +10,8 @@ uk:
           applicant2SignInDivorceUrl: ${NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DIVORCE_URL:https://nfdiv-apply-for-divorce.aat.platform.hmcts.net/applicant2}
           signInDissolutionUrl: ${NOTIFY_TEMPLATE_SIGN_IN_DISSOLUTION_URL:https://nfdiv-end-civil-partnership.aat.platform.hmcts.net/}
           applicant2SignInDissolutionUrl: ${NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DISSOLUTION_URL:https://nfdiv-end-civil-partnership.aat.platform.hmcts.net/applicant2}
+          switchToSoleApplicationDivorceUrl: ${NOTIFY_TEMPLATE_SWITCH_TO_SOLE_APPLICATION_URL:https://nfdiv-apply-for-divorce.aat.platform.hmcts.net/switch-to-sole-application}
+          switchToSoleApplicationDissolutionUrl: ${NOTIFY_TEMPLATE_SWITCH_TO_SOLE_APPLICATION_URL:https://nfdiv-end-civil-partnership.aat.platform.hmcts.net/switch-to-sole-application}
```

Comment:
both are using NOTIFY_TEMPLATE_SWITCH_TO_SOLE_APPLICATION_URL env var but values are different should be have two env vars ?

Reply:
Will update, must have missed that!